src/content/app/genome-browser/components/track-panel/components/track-panel-list/track-panel-items/TrackPanelTranscript.tsx

```diff
@@ -67,7 +67,9 @@ const TrackPanelTranscript = (props: Props) => {
     </span>
   ) : (
     <span className={styles.labelTextSecondary}>
-      {transcript.metadata.biotype.label}
+      {getTranscriptSupportLevel(transcript)?.label
+        ? getTranscriptSupportLevel(transcript)?.label
+        : transcript.metadata.biotype.label}
```

Comment:
Another way to write this would be:

```
{getTranscriptSupportLevel(transcript)?.label || transcript.metadata.biotype.label}
```

I don't have a strong opinion on this one as both will work fine. So upto you whether to change it or not :)

Reply:
Agreed. i have change it to your suggestion, code is cleaner and simpler. Have push the changes. If you could review please.

Thanks